soc/arm/st_stm32/stm32mp1/dts_fixup.h

```diff
@@ -195,6 +195,8 @@
 #define DT_GPIO_STM32_GPIOK_CLOCK_BUS           \
 				DT_ST_STM32_GPIO_5000C000_CLOCK_BUS
 
+#define DT_IPM_DEV		DT_ST_STM32_IPCC_MAILBOX_4C001000_LABEL
```

Comment:
I want to avoid adding DT_IPM_DEV, see what I did in PR #17553 w/the Kconfig and such to get the params from the device-tree.